int32 pointID
---
bool success